lib/services/database/mock_firestore.dart

```diff
@@ -1,6 +1,23 @@
+import 'dart:async';
+import 'package:veggie_go_malaysia/datamodels/announcement.dart';
 import 'package:veggie_go_malaysia/datamodels/restaurant.dart';
 import 'package:veggie_go_malaysia/services/database/abstract_firestore.dart';
 
 class MockFirestore implements DatabaseService {
-  Future<List<Restaurant>> _fetchResults;
+
+  Future<List<Announcement>> getAnnouncements() async {
+    return [
+      Announcement(
```

Comment:
Announcement Widget needs some ui fix for the Text, as the text doesn't scale properly according to the carousel container size

![Image from iOS](https://user-images.githubusercontent.com/13532268/83940031-21e9bc80-a814-11ea-8ff9-a0275f973157.jpg)